flake8_broken_line.py

```diff
@@ -38,7 +40,7 @@ def check_line_breaks(
         if line_token.start[0] in reported:
             continue  # There might be several tokens on a same line.
 
-        if _INVALID_LINE_BREAK.search(line_token.line):
+        if all(condition.search(line_token.line) for condition in CONDITIONS):
```

Comment:
Using this pattern since the combination of flakes for LineBreak after operator + Line break before operator + 80 line length made formatting this line impossible